UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+
+Removed variables:
+- `--pf-c-backdrop--BackdropFilter`
+
+### Breadcrumb
+- Added word-break, display link and icon inline, move divider [(#2916)](https://github.com/patternfly/patternfly/pull/2916)
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Made redhat font default, add opt-in for overpass [(#2955)](https://github.com/patternfly/patternfly/pull/2955)
+
+Updated markup:
+- Any reference to the divider element `.pf-c-breadcrumb__item-divider` will need to be moved in the markup to come before the breadcrumb link element `.pf-c-breadcrumb__link` instead of after it.
+
+Updated CSS:
+- The text will now break if there is a long string in an item link that is wider than the viewport.
+- The divider will always appear inline with the item text instead of wrapping to a new line in some cases.
+
+Removed classes:
+- `.pf-m-hover` from `.pf-c-breadcrumb__link`
+  - The `:hover` selector still has styles applied to it.
+
+Renamed variables:
+- `--pf-c-breadcrumb__item-divider--MarginLeft` to `--pf-c-breadcrumb__item-divider--MarginRight`
+
+Removed variables:
+- `--pf-c-breadcrumb__item--FontWeight`
+- `--pf-c-breadcrumb__link--FontWeight`
+
+### Button
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Cleaned up vars [(#3028)](https://github.com/patternfly/patternfly/pull/3028)
+- Updated control variation disabled state [(#3049)](https://github.com/patternfly/patternfly/pull/3049)
+- Added modifiers to position icon in button [(#2828)](https://github.com/patternfly/patternfly/pull/2828)
+- Made shield styles optional by default [(#2872)](https://github.com/patternfly/patternfly/pull/2872)
+
+Updated markup:
+- Adds the modifiers `.pf-m-start` and `.pf-m-end` to position `pf-c-button__icon` in the button component depending on whether it comes before or after text. If you are using a button with an icon, you will need to use one of these modifiers to style the icon.
+
+Updated CSS:
+- Link and plain buttons now define `background-color: transparent` for their normal, hover, focus, and active states.
+- The control button now sets a white background and default text color for its normal, hover, focus, and active states.
+- The `:active` CSS selectors in the button component's stylesheet were moved after `:focus`, so that if the `:focus` and `:active` pseudo selectors declare styles on the same CSS properties for an element, the `:active` styles will now be visible when the element is `:active`. No changes are needed to consume this change, however any `:active` styles that may have been overridden by the `:focus` styles coming later in the stylesheet may now be visible.
+
+Removed classes:
+- `.pf-c-button__text` - element was removed
+- `.pf-m-hover` from `.pf-c-button`
+- `.pf-m-focus` from `.pf-c-button`
+  - The `:hover` and `:focus` selectors still have styles applied to them.
```

Comment:
I recommend making 282 and 283 consistent in structure with the other line elements. It isn't clear what they mean, especially since we say "Removed classes." Maybe we mean "Changed `.pf-m-hover` to `.pf-c-button` " or "Removed `.pf-m-hover` from `.pf-c-button` " ?

Reply:
Those lines are meant to communicate "Removed .pf-m-hover/pf-m-focus from .pf-c-button"